release-1.8/release_notes_draft.md

```diff
@@ -221,3 +221,24 @@ kind.
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
 ## External Dependencies
+Continuous integration builds have used the following versions of external
+dependencies, however, this is not a strong recommendation and users should
+consult an appropriate installation or upgrade guide before deciding what
+versions of etcd, docker or rkt to use.
```

Comment:
Continuous integration builds have used Docker versions 1.11.2, 1.12.6, and 1.13.1 (17.03.2). These versions have been validated on Kubernetes 1.8. However, consult an appropriate installation or upgrade guide before deciding what versions of docker to use.

Reply:
Question: In this section, we are not talking about etcd or rkt. So, can we remove those references and focus only on docker ? cc @calebamiles

Reply:
I have them removed. But whoever works on that should add the appropriate information here.